addons/bus/static/tests/helpers/mock_python_environment.js

```diff
@@ -92,6 +94,12 @@ export const pyEnvTarget = {
             throw new Error("Unauthorized");
         }
         this.cookie.set("sid", user.id);
+        const userSession = {
+            name: user.name,
+            partnerId: user.partner_id,
+            userId: user.id,
+        };
+        patchUserWithCleanup(userSession);
```

Comment:
Note: session needs to be sync with pyEnv when using login/logout, patching user only in start is not enough